proxy/src/main/java/com/wavefront/agent/listeners/tracing/JaegerThriftCollectorHandler.java

```diff
@@ -382,6 +384,16 @@ private void processSpan(io.jaegertracing.thriftjava.Span span,
     }
   }
 
+  private boolean sample(Span wavefrontSpan) {
+    if (sampler != null && sampler.sample(wavefrontSpan.getName(),
```

Comment:
no need to check for sampler is null since it is not marked Nullable
also if at all sampler was null then default is true and not false